core/data-api/src/main/java/com/yapp/data_api/MyClass.kt

```
@@ -0,0 +1,4 @@
+package com.yapp.data_api
+
+class MyClass  {
+}
```

detekt (1.23.7)

[warning] The class or object MyClass is empty.

(detekt.empty-blocks.EmptyClassBlock)